Update Readme

vxsh4d0w opened the pull request:
Just few suggestions

<!--- IMPORTANT: Please review [how to contribute](../CONTRIBUTING.md) before proceeding further. -->
<!--- IMPORTANT: If this is a Work in Progress PR, please mark it as such in GitHub. -->

## Description

<!--- Describe in detail the proposed mods -->

This PR tackles:

- Translation review
- ...
- ...

In particular, the proprosed changes are related the content of this readme section.

## Checklist

<!--- Please insert an ‘x’ after you complete each step -->

- [x] I have followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md).
- [] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [ ] I have written new tests for my core changes, as applicable.
- [ ] I have successfully run tests with my changes locally.
- [x] It is ready for review! :rocket:

## Fixes

<!-- Please insert the issue numbers after the # symbol -->

- Fixes #ISSUE_NUMBER

vxsh4d0w committed:
Update Readme

Just few suggestions

ImmuniDGR commented:
🚔 **Safety Check** 🚔

🔰 **Result** 🔰


✅ All configuration files are in line with the master branch.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #1045 (id: e6c4df26-cb15-48eb-a77e-72d2bebf5a02)
- Last verified commit: 83b6ee1845787be8488b462758f14473d45476b1
- Time of check: 02/06/2020, 13:30:49 UTC
- The following protected files have been checked for changes: .editorconfig, .gitmodules, Gemfile, Gemfile.lock, app/build.gradle, app/proguard-rules.pro, build.gradle, ci/changelog\_config.tpl, ci/danger/commitlint/index.ts, ci/danger/commitlint/rules.ts, ci/danger/danger.ts, ci/danger/ktlint.ts, dangerfile.ts, fastlane/Appfile, fastlane/Fastfile, gradle.properties, gradlew, gradlew.bat, package.json, publiccode.yml, scheduler\_config.json, settings.gradle, template.properties, yarn.lock.

ImmuniDGR commented:
<!--
  1 failure:  Commitlint failed...
  0 warning: 
  4 messages
  
  DangerID: danger-id-Danger;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Commitlint failed on the following commits (please do rebase): 

 - [83b6ee1845787be8488b462758f14473d45476b1] Update Readme

Just few suggestions
	- subject may not be empty
	- type may not be empty

Look at our rules and the [Conventional Commits](https://www.conventionalcommits.org) guidelines.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Thank you for submitting a pull request! The team will review your submission as soon as possible.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 83b6ee1845787be8488b462758f14473d45476b1
</p>

enricocid commented:
Secondo me suona meglio:
"Further information about the project can be found under the following documents:"

Inutile ripetere Immuni due volte (toglierei anche about the project se per questo), imo. Giusto un consiglio :-)

vxsh4d0w committed:
Typo error fix

According a suggestion and after realising my typo error, I added a couple of changes.

vxsh4d0w commented:
> under the

under != all'interno di....

Sure,

enricocid commented:
> > under the
> 
> under != all'interno di....
> 
> Sure, 

Corretto con within, il mio suggerimento è ancora valido

vxsh4d0w committed:
New changes

ilovelinux commented on README.md:
```diff
@@ -142,15 +143,16 @@ To run the instrumentation/UI tests, execute the following Gradle task:
+In addition, to make the code open-source, we support people to verify that every build published on the App Store comes from a new commit stored in this repository. Please refer to the [Immuni Technology Description](https://github.com/immuni-app/documentation/blob/master/Technology%20Description.md#android-app-technologies) for a complete overview of the goals and status of this effort.
 
-Currently, we have a working open continuous integration for building the client. [Here](.circleci/config.yml) is the full specification. When it comes to reproducible builds, we will instead open an issue explaining what we have done so far and any missing steps.
+Currently we provide an open and continuous integration for building and improving the client. [Here](.circleci/config.yml) is the full specification. Every new release will include, reported as a new Issue, what we have done and what we suppose to improve and implement.
```
Can someone confirm the last part of this translation is/isn't ok? It doesn't look right to me

ilovelinux commented on README.md:
```diff
@@ -142,15 +143,16 @@ To run the instrumentation/UI tests, execute the following Gradle task:
 # Contributing
 
-Contributions are most welcome. Before proceeding, please read the [Code of Conduct](CODE_OF_CONDUCT.md) for guidance on how to approach the community and create a positive environment. Additionally, please read our [CONTRIBUTING](CONTRIBUTING.md) file, which contains guidance on ensuring a smooth contribution process.
+Contributions are really appreciated and more than welcome. Before proceeding, please read carefully the [Code of Conduct](CODE_OF_CONDUCT.md) and consider it as an "How-To" guide to contribute the community and create a positive and collaborative environment. Additionally, please read also our [CONTRIBUTING](CONTRIBUTING.md) file that provides all info to make the contribution process easy.
```
```suggestion
Contributions are welcome. Before proceeding, please read carefully the [Code of Conduct](CODE_OF_CONDUCT.md) and consider it as an "How-To" guide to contribute and have a positive approach to the community in order to create a collaborative environment. Additionally, please read also our [CONTRIBUTING](CONTRIBUTING.md) file that provides all info to make the contribution process easy.
```

ilovelinux commented on README.md:
```diff
@@ -142,15 +143,16 @@ To run the instrumentation/UI tests, execute the following Gradle task:
+Contributions are really appreciated and more than welcome. Before proceeding, please read carefully the [Code of Conduct](CODE_OF_CONDUCT.md) and consider it as an "How-To" guide to contribute the community and create a positive and collaborative environment. Additionally, please read also our [CONTRIBUTING](CONTRIBUTING.md) file that provides all info to make the contribution process easy.
 
-The Immuni project is composed of different repositories—one for each component or service. Please use this repository for contributions strictly relevant to the Immuni Android client. To propose a feature request, please open an issue in the [Documentation repository](https://github.com/immuni-app/documentation). This lets everyone involved see it, consider it, and participate in the discussion. ...
+The Immuni project consists of several different repositories (one per component or service). Please consider this repository only related to Immuni for iOS. To propose or request a new feature, please open an issue here [Documentation repository](https://github.com/immuni-app/immuni-documentation). 
```
```suggestion
The Immuni project consists of several different repositories (one per component or service). Please consider this repository only related to Immuni for Android. To propose a new feature, please open an issue here [Documentation repository](https://github.com/immuni-app/immuni-documentation).
```

ilovelinux commented on README.md:
```diff
@@ -142,15 +143,16 @@ To run the instrumentation/UI tests, execute the following Gradle task:
 
-The Immuni project is composed of different repositories—one for each component or service. Please use this repository for contributions strictly relevant to the Immuni Android client. To propose a feature request, please open an issue in the [Documentation repository](https://github.com/immuni-app/documentation). This lets everyone involved see it, consider it, and participate in the discussion. ...
+The Immuni project consists of several different repositories (one per component or service). Please consider this repository only related to Immuni for iOS. To propose or request a new feature, please open an issue here [Documentation repository](https://github.com/immuni-app/immuni-documentation). 
+This will make everyone feel involved to see it, consider it, and participate to the discussion. Any opened issue or pulled request could slow down the overall process.
```
```suggestion
This will make everyone feel involved to see it, consider it, and participate to the discussion. Any open issue or pull request could slow down the overall process.
```
The last phrase is confusing and is actually discussed #300
